src/Features/CSharp/Portable/IntroduceVariable/CSharpIntroduceVariableService_IntroduceLocal.cs

```diff
@@ -53,7 +53,7 @@ internal partial class CSharpIntroduceVariableService
             var text = await document.Document.GetTextAsync(cancellationToken).ConfigureAwait(false);
             if (!text.AreOnSameLine(containerToGenerateInto.GetFirstToken(), containerToGenerateInto.GetLastToken()))
             {
-                declarationStatement = declarationStatement.WithAppendedTrailingTrivia(SyntaxFactory.ElasticCarriageReturnLineFeed);
+                declarationStatement = declarationStatement.WithTrailingTrivia(SyntaxFactory.CarriageReturnLineFeed);
```

Comment:
i'm a little confused why the elastic crlf wasn't appropriate here.  why did that not work?